src/components/landing/BenefitsCarousel.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+// Import Swiper React components
+import { Swiper, SwiperSlide } from "swiper/react";
+
+// Import Swiper styles
+import "swiper/css";
+import "swiper/css/pagination";
+
+import "../../styles/landing/benefitsCarousel.css";
+
+// import required modules
+import { EffectCoverflow, Navigation, Pagination } from "swiper/modules";
+
+const BenefitsCarousel = ({ slides }) => {
+  const [_active, _setActive] = useState(0);
+  const [_activeIndex, setActiveIndex] = useState(0);
+
+  const [showPrevButton, setShowPrevButton] = useState(false);
+  const [showNextButton, setShowNextButton] = useState(true);
+
+  const handleSlideChange = (swiper) => {
+    setActiveIndex(swiper.activeIndex);
+    setShowPrevButton(!swiper.isBeginning);
+    setShowNextButton(!swiper.isEnd);
+  };
+
+  const colors = ["#EDFCE2", "#EAE2FC", "#FCF6E2", "#EDF2FE"];
```

Comment:
Colors can be moved to the parent component and be passed here as props - the same colors are duplicated and hardcoded in [Carousel.jsx](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2899/files#diff-15dc7ce7ad4499b67dadfc25fd4552255997a98a5aac4d5cd546b15bfca8e084R22), so the parent can pass them as a prop to both components